tests/unit/fem/test_linearform_ext.cpp

```diff
@@ -282,4 +282,33 @@ TEST_CASE("Linear Form Extension", "[LinearFormExtension], [CUDA]")
 
       REQUIRE(d1.Norml2() == MFEM_Approx(0.0));
    }
+
+   SECTION("VectorFE")
+   {
+      Mesh mesh(mesh_file);
+      const int dim = mesh.Dimension();
+
+      CAPTURE(mesh_file, dim, p);
+
+      RT_FECollection fec(p-1, dim);
```

Comment:
Should ND be tested as well?

Reply:
ND is tested in `test_face_restriction.cpp`. Only RT is tested in the LinearFormExtension test because we have not implemented device versions of any ND-compatible linear forms (`VectorFEBoundaryFluxLFIntegrator` only makes sense for RT spaces)